### ISSUE TITLE
[repository schema] Remove (initially: add) repository attribute for application version (initially: extension) ID

### ISSUE THREAD

**Issue by kleihan** [FIXTradingCommunity/fix-orchestra#186]
Orchestra has a number of attributes for the `<repository>` element. One of them (`applVerId`) is for the *application* version (not to be confused with the *repository* version attribute `version`). It is proposed to add a new attribute `applExtId` to fully support FIX versioning covered by the FIX fields ApplVerID(1128) and ApplExtID(1156) in the StandardHeader component.
```xml
<xs:attribute name="applVerId" type="xs:string"/>
<xs:attribute name="applExtId" type="xs:integer"/>
```

**Comment by kleihan** [comment]
This is linked to the existing issue https://github.com/FIXTradingCommunity/fix-orchestra-spec/issues/46 about DC terms to use. That may be a better approach than adding attributes to the root element.

**Comment by kleihan** [comment]
The `repository` element has a generic `version` attribute for the schema. Rather than having explicit attributes for the application, the generic DC terms should cover this information, see FIXTradingCommunity/fix-orchestra-spec#46.

Proposal is now to remove the existing `applVerId` attribute instead of adding a second, more granular one.
